docs/contribute/processes/release.md

```diff
@@ -2,681 +2,163 @@
-3.  move mouse until new key is generated
-4.  press Save private key and choose a filename to save private key on
-    disk (ignore warning of empty passphrase)
-5.  copy public key into the clipboard
-6.  log in on [Sourceforge](https://sourceforge.net/account/login.php)
-7.  go to [SSH Keys](https://sourceforge.net/account/ssh)
-8.  paste your public key into the area labeled Authorized keys
-9.  click Update
-
-### How to make a re-release
-
-*   Sometimes a critical bug is shipped with a release that just
-    happened, that makes it necessary to re-release.
-*   Since making a release is a lot of work and also is confusing to the
-    users, we should use the following criteria to determine whether a
-    re-release is appropriate:
-    *   Is the release unusable for users due to this bug?
-    *   Does the bug represent a regression?
-    *   Have less than 72 hours passed since the release?
-*   These rules can be bend at will, but it might also make sense to
-    just withdraw the release, by updating site.xml
-
-*   How a re-release is done:
-    *   The first rule is:
-        *   A release is always made from the release branch
-    *   To achieve this, remember:
-        *   Fixes go to the release branch
-        *   Features go to trunk
-    *   Once the release branch has been patched and fixed, the release
-        is made from the branch. Afterwards the branch is merged to
-        trunk to integrate the fixes to trunk.
-    *   If another bug is found, repeat:
-        *   Fixed go to the release branch
-        *   Features stay in trunk
-    *   The second rule is:
-        *   Don't forget to merge back.
-
-### Updating Dependencies
-
-1.  The JARs should have their version numbers in the name.
-2.  Make sure you have copied the new JARs to
-    &lt;rootofsarosproject&gt;/lib/&lt;/rootofsarosproject&gt;
-3.  right click on your Saros project in eclipse, choose "Build
-    Path"-&gt;"Configure Build Path.."
-4.  click tab "Libraries"
-5.  click button "Add JARs" to add new JARs to the project
-6.  click "Ok" and you are ready, maybe you have to rebuild the project
-7.  update all relevant information (classpath, build properties, etc.)
-    in the plugin.xml
+
+## When do we release?
+
+We publish releases when a relevant set of changes is integrated in the master branch and the team wants to release.
+
+## How do I create the release artifacts?
+In the following the task which are required to build all release artifacts are described.
+
+Please make sure before creating the artifacts that **all unit and stf tests are successful.**
+
+### Eclipse
+
+**Artifacts:**
+* Update Site
+* Drop-in
+
+#### Update Site
+
+* Open an eclipse instance which contains a working [Saros development setup](https://www.saros-project.org/contribute/development-environment.html)
+* Open the project `Saros Update Site`
+* Open the file `site.xml`
+* Make sure that the category `DPP` contains the current feature `saros.feature*`
```

Comment:
Is this really still `DPP`? How much work is it to change this?

Reply:
Changing the category should be simply, but we have to check whether the update still works as expected (I assume that it works, but we should test it). The question is whether we change the name or remove the category (a category contains multiple features, but we have just one feature (saros.eclipse)). If you don't use a category, the update-site (during the installation process) shows the category `Unknown` (or something similar). I open a separate issue in order to track this.

Reply:
See #557